.github/workflows/ruby.yml

```diff
@@ -21,6 +21,10 @@ jobs:
     strategy:
       matrix:
         ruby: [2.7, '3.0']
+        faraday_version: ['']
+        include:
+          - ruby: 2.7
+            faraday_version: '~> 1.0'
```

Comment:
suggestion: Could we explicitly declare Faraday '~> 2.0' for ruby 3.0? It would make the config easier to read and understand.

Reply:
You mean instead of the `['']`? I considered that and was concerned that the faraday version default would then need to be changed here and in `Gemfile.extra`. Am I overthinking it?

Reply:
Also this isn't making Faraday 1 for 2.7, it's adding a third matrix item which is ruby 2.7 and faraday 1.

Reply:
I see. I was confused as well. I think this is fine.